.github/workflows/_test.yml

```diff
@@ -28,7 +28,12 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
-    name: "make test #${{ matrix.python-version }}"
+        sqlalchemy-version:
+          - "latest"
+        include:
+          - sqlalchemy-version: "1.4"
```

Comment:
Potentially fragile since it assumes that the lock file has sqlalchemy 2?

Reply:
Maybe rewording the default for `sqlalchemy-version` from "latest" to "default" or "from-lock-file" would make it more acceptable by making it explicit?

If not, I can specify both ">2" and "~=1.4" and adjust the logic for not polluting the name needlessly with matching against ">2".

Do let me know what change if any you prefer :slightly_smiling_face:

Reply:
@krassowski assigning to @efriis since he's been setting up all the devs ops

Reply:
Just checking back here one week later :)